src/main/scala/io/epiphanous/flinkrunner/model/sink/JdbcSinkConfig.scala

```diff
@@ -177,6 +177,10 @@ case class JdbcSinkConfig[ADT <: FlinkEvent](
     case _             => Seq.empty
   }
 
+  val isTimescale: Boolean = config
```

Comment:
Instead of a boolean `is.timescale` option and several other options at the `table` level, can we have a unified `timescale` config block under `table`, and `isTimescale` is computed by the existence of the timescale config block:

```
table {
  name = ...
  timescale {
    time.column = ...
    number.partitions = ...
    partitioning.column = ....
    chunk.time.interval = ...
  }
}
```